wip-0022.md

```diff
@@ -109,12 +101,14 @@ Due to this last point, this MUST be considered a consensus-critical protocol im
 
 ## Reference Implementation
 
-A reference implementation for the proposed protocol improvement can be found in the [pull request #2229](https://github.com/witnet/witnet-rust/pull/2229) of the [witnet-rust](https://github.com/witnet/witnet-rust/) repository.
+A reference implementation for this protocol improvement was proposed in [pull request #2229](https://github.com/witnet/witnet-rust/pull/2229) and has been further refined in [pull request #2337](https://github.com/witnet/witnet-rust/pull/2337) of the [witnet-rust](https://github.com/witnet/witnet-rust/) repository.
 
 ## Adoption Plan
 
-To be determined.
+An activation date is proposed for February 21 2023 at 9am UTC, that is, protocol epoch #1651280.
+
+From the perspective of TAPI, the signaling bit is 3 (0b00000000000000000000000000001000) and the first_signaling_epoch is set to #1651280.
```

Comment:
This is not the activation date but the date in which the signaling begins. I just noticed that, yes, every other WIP  after 0014 is wrong.

```suggestion
An activation date is proposed for March 7 2023 at 9am UTC, that is, protocol epoch #1678160.

From the perspective of TAPI, the signaling bit is 3 (0b00000000000000000000000000001000) and the first_signaling_epoch is set to #1651280 (February 21 at 9am UTC).
```